src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,57 @@
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { useAppSelector } from '../../app/hooks';
+
+type Props = {
+  todo: Todo;
+  onSelect: (todo: Todo) => void;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo, onSelect }) => {
+  const { id, title, completed } = todo;
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const handleSelectTodo = () => {
+    onSelect(todo);
+  };
+
+  return (
+    <tr data-cy="todo">
+      <td className="is-vcentered">{id}</td>
+      <td className="is-vcentered">
+        {completed ? (
+          <span className="icon" data-cy="iconCompleted">
+            <i className="fas fa-check" />
+          </span>
+        ) : (
+          ''
+        )}
+      </td>
+
+      <td className="is-vcentered is-expanded">
+        <p
```

Comment:
The use of the 'cn' library here seems unnecessary. You could directly write the ternary operator inside the className property. It would make the code cleaner and easier to read.